schemas/network.json

```diff
@@ -0,0 +1,199 @@
+{
+  "$id": "https://fildev.network/schemas/network.json",
+  "type": "object",
+  "properties": {
+    "Name": {
+      "type": "string",
+      "examples": [
+        "calibration"
+      ]
+    },
+    "Purpose": {
```

Comment:
Should we also have a field for "maintainer" where type is "string" and examples could be "@travisperson" or "Protocol Labs" or "Company Y". Would allow us to include networks here that are maintained by non-PL entities eventually (I fully anticipate these will crop up down the road!).

Reply:
Sure